### DISCUSSION TITLE
Mega2560 - Restore default 16u2 configuration

### DISCUSSION THREAD

**Discussion by SciFi-Bob** [NicoHood/HoodLoader2#136] [Q&A]
Hi there,  I am really hoping that someone can help me out of a days long spiral of fails :-)

I have 2 Mega 2560 boards that I have had for a while, both have Hoodloader on and also I have a Leonardo that I used to install HL on both the Mega's

I am looking to revert the two Mega's back to their factory standard but for the life of me I can't seem to get it to work!

In the Installation Sketch I have the 202 firmware selected and I get the output as follows.

`Attempting to enter programming mode ...
Entered programming mode OK.
Signature = 0x1E 0x94 0x89 
Processor = ATmega16U2
Flash memory size = 16384 bytes.
LFuse = 0xEF 
HFuse = 0xD9 
EFuse = 0xF4 
Lock byte = 0xCF 
Clock calibration = 0x58 
No bootloader support for this device.
Programming mode off.

10
9`

If I select the number 4 Hootloader option the devices program successfully.

I am sure that it's something stupid I am missing but I have been going round and round in circles.

Thanks in advance, if there is additional information I am missing out on please let me know.

**Comment by Mcletsplay001** [comment]
hey i had the same problem with my mega had to use the 302 and then I could use 202 firmware. hope it helps
